lib/gnarails/cli/application.rb

```diff
@@ -5,153 +5,14 @@ module Cli
-
-      method_option :skip_action_mailer,
-        type: :boolean,
-        aliases: "-M",
-        default: false,
-        desc: "Skip Action Mailer files"
-
-      method_option :skip_active_record,
-        type: :boolean,
-        aliases: "-O",
-        default: false,
-        desc: "Skip Active Record files"
-
-      method_option :skip_active_storage,
-        type: :boolean,
-        default: false,
-        desc: "Skip Active Storage files"
-
-      method_option :skip_puma,
-        type: :boolean,
-        aliases: "-P",
-        default: false,
-        desc: "Skip Puma related files"
-
-      method_option :skip_action_cable,
-        type: :boolean,
-        aliases: "-C",
-        default: false,
-        desc: "Skip Action Cable files"
-
-      method_option :skip_sprockets,
-        type: :boolean,
-        aliases: "-S",
-        default: false,
-        desc: "Skip Sprockets files"
-
-      method_option :skip_spring,
-        type: :boolean,
-        default: false,
-        desc: "Don't install Spring application preloader"
-
-      method_option :skip_listen,
-        type: :boolean,
-        default: false,
-        desc: "Don't generate configuration that depends on the listen gem"
+      attr_reader :args
 
-      method_option :skip_coffee,
-        type: :boolean,
-        default: false,
-        desc: "Don't use CoffeeScript"
-
-      method_option :skip_javascript,
-        type: :boolean,
-        aliases: "-J",
-        default: false,
-        desc: "Skip JavaScript files"
-
-      method_option :skip_turbolinks,
-        type: :boolean,
-        default: false,
-        desc: "Skip turbolinks gem"
-
-      method_option :skip_test,
-        type: :boolean,
-        aliases: "-T",
-        default: false,
-        desc: "Skip test files"
-
-      method_option :skip_system_test,
-        type: :boolean,
-        default: false,
-        desc: "Skip system test files"
-
-      method_option :skip_bootsnap,
-        type: :boolean,
-        default: false,
-        desc: "Skip bootsnap gem"
-
-      method_option :dev,
-        type: :boolean,
-        default: false,
-        desc: "Setup the application with Gemfile pointing to your Rails checkout"
-
-      method_option :edge,
-        type: :boolean,
-        default: false,
-        desc: "Setup the application with Gemfile pointing to Rails repository"
-
-      method_option :rc,
-        type: :string,
-        default: nil,
-        desc: "Path to file containing extra configuration options for rails command"
-
-      method_option :no_rc,
-        type: :boolean,
-        default: false,
-        desc: "Skip loading of extra configuration options from .railsrc file"
-
-      method_option :api,
-        type: :boolean,
-        desc: "Preconfigure smaller stack for API only apps"
-
-      method_option :skip_bundle,
-        type: :boolean,
-        aliases: "-B",
-        default: false,
-        desc: "Don't run bundle install"
```

Comment:
For the record, I'm very much in favor of not having to play this cat and mouse game, particularly when we're not wanting to be in the business of supporting multiple underlying versions of rails that folks could be using along with gnarails.